src/feedback/trulens/feedback/groundtruth.py

```diff
@@ -193,10 +196,7 @@ def agreement_measure(
                 prompt, response, ground_truth_response
             )
             ret = (
-                re_configured_rating(
-                    agreement_txt, min_score_val=0, max_score_val=3
-                )
-                / 3,
+                re_0_10_rating(agreement_txt) / 10,
```

Comment:
This is b/c the instructions in the agreement prompt (for  feedback function `GroundtruthAgreement.agreement_measure`) hasn't been updated to take configurable output score space yet.

I will do that along with several other feedback prompts in a separate PR, and this change is here so that the e2e notebook can run successfully